hadoop-hdds/framework/src/test/java/org/apache/hadoop/hdds/server/events/TestEventWatcher.java

```diff
@@ -179,22 +179,32 @@ public void testMetrics() throws InterruptedException {
 
     queue.fireEvent(REPLICATION_COMPLETED, event1Completed);
 
+    //lease manager timeout = 2000L
```

Comment:
May want to increase this even further, say 2x or 3x of LM timeout. 200ms of buffer is too low.